src/main/scala/scalafix/sbt/ScalafixFailed.scala

```diff
@@ -0,0 +1,8 @@
+package scalafix.sbt
+
+import sbt.FeedbackProvidedException
```

Comment:
cc/ @dwijnand 

Why is `sbt.internal.util.FeedbackProvidedException` exposed in the public sbt API via `sbt.FeedbackProvidedException`? Type aliases don't exist in bytecode so
```scala
type FeedbackProvidedException = sbt.internal.util.FeedbackProvidedException
```
will make sbt-scalafix binaries reference `sbt.internal` classes.

Also, why is `FeedbackProvidedException` internal? It's a useful helper for writing custom sbt tasks.

Reply:
My understanding of the general convention around `internal` packages is that binaries have no guarantees about binary stability. Removing `sbt.internal.util.FeedbackProvidedException` and pointing `sbt.FeedbackProvidedException` somewhere else is a source compatible but binary breaking change.

Reply:
sbt has done a very poor job in this regard. For instance the entirety of sbt's parser/complete API is "internal" but clearly can't break.

This is just another example of that. TIL `sbt.FeedbackProvidedException`.

Reply:
Fair enough, too late to change now :)